pmoves/libs/providers/embedding.py

```
@@ -51,19 +53,27 @@ def _embed_tensorzero(text: str):
 def _embed_ollama(text: str):
     try:
         r = requests.post(f"{OLLAMA_URL}/api/embeddings", json={"model": OLLAMA_EMBED_MODEL, "prompt": text}, timeout=15)
         if r.ok:
             v = r.json().get("embedding")
-            if v: return v
+            if v:
+                logger.info("Ollama embedding OK: model=%s dim=%d", OLLAMA_EMBED_MODEL, len(v))
+                return v
+        else:
+            logger.warning("Ollama embedding failed: %s", r.status_code)
     except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)